pkgs/os-specific/linux/uvcdynctrl/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libxml2 }:
+
+stdenv.mkDerivation rec {
+  version = "0.2";
```

Comment:
Looks like version for this rev is `0.3.0`? https://github.com/cshorler/webcam-tools/blob/bee2ef3c9e350fd859f08cd0e6745871e5f55cb9/uvcdynctrl/CMakeLists.txt#L18